docs/examples/basic.md

```diff
@@ -0,0 +1,88 @@
+# Basic OTG Configurations
+
+## Single TCP Traffic Flow between two Test Ports
+
+File: [`flow.otg.yaml`](flow.otg.yaml).
+
+This single flow configuration specifies:
+
+* Flow Name: `f1`
+* Packet Schedule: 
+    * `1000` packets 
+    * of `512B` size 
+    * to be transmitted as fixed rate of `100pps`
+* Flow Template: 
+    * Ethernet src `02:00:00:00:01:aa` dst `02:00:00:00:02:aa`
+    * IPv4 src `192.0.2.1` dst `192.0.2.2`
+    * TCP ports src `1024 x 7 x increment` dst `7`
```

Comment:
`1024 x 7 x increment` is a bit confusing for me.  What about `start at 1024 increment by 7`?

Reply:
good suggestion, thanks!